sorc/ncep_post.fd/INITPOST_NETCDF.f

```diff
@@ -45,7 +45,8 @@ SUBROUTINE INITPOST_NETCDF(ncid3d)
               o3vdiff, o3prod, o3tndy, mwpv, unknown, vdiffzacce, zgdrag,cnvctummixing,         &
               vdiffmacce, mgdrag, cnvctvmmixing, ncnvctcfrac, cnvctumflx, cnvctdmflx,           &
               cnvctzgdrag, sconvmois, cnvctmgdrag, cnvctdetmflx, duwt, duem, dusd, dudp,        &
-              wh, qqg, ref_10cm
+              wh, qqg, ref_10cm, pmtf, o3mr, ozcon
```

Comment:
@hsinmulin-NOAA The new variable o3mr is not used. Can you remove it in this routine and all related routines?